src/MessagingAvatar/MessagingAvatar.tsx

```diff
@@ -39,9 +39,8 @@ export const MessagingAvatar: React.FC<Props> = ({ className, text, color, image
     <div
       className={classNames(cssClasses.CIRCLE, className)}
       // If defined, use color. Otherwise, use seed to determine color.
-      // If all else fails, we can use text as the seed to determine color.
       style={{
-        backgroundColor: color.color || `#${_determineAvatarColor(color.seed || text)}`,
```

Comment:
Would there be benefit to leave this for a situation where `color.seed = ""?`

Reply:
I agree this seems unnecessary since we always require providing the color or seed

Reply:
Oh just re-read your comment, good point that it could technically be an empty string. Technically, so could `text`, but that seems unlikely. I could go either way on this, providing an empty string for seed shouldn't really be allowed but it's not terrible to have a fallback if someone does.

Reply:
That's a good point, that `text` could be empty too. If `seed` is empty, it defaults to a light blue based on the `determineAvatarColor` function, so I think that's sufficient rather than relying on `text`. I'll go with this change I've made. Thanks!